cluster/terraform_kubernetes/ingress_controller.tf

```diff
@@ -54,7 +54,7 @@ resource "helm_release" "ingress-nginx" {
   }
   set {
     name  = "controller.config.proxy-buffer-size"
-    value = "8k"
+    value = "24k"
```

Comment:
Could you add a comment with a description and maybe a link to the property?

Reply:
added comment